services/user.service.ts

```diff
@@ -233,29 +239,23 @@ export async function editPasswordService(userId, { currentPassword, newPassword
 
 
 // Refresh Token을 이용해 Access Token 재발급
-export async function refreshService(refreshToken) {
+export async function refreshService(refreshToken: any): Promise<string>  {
```

Comment:
typeScript에서는 any타입을 쓰시면 안됩니다.
적절한 타입으로 고쳐주세요~!

Reply:
다음과 같이 string 타입으로 변경했습니다!
```suggestion
export async function refreshService(refreshToken: string): Promise<string>  {   
```